cfgov/unprocessed/css/molecules/archived-pill.less

```diff
@@ -0,0 +1,33 @@
+/* topdoc
+  name: Archived pill
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-archived-pill">
+            Archived
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-archived-pill
+  tags:
+    - cfgov-molecules
+*/
+
+.m-archived-pill {
+    padding: unit( 9px / @base-font-size-px, em ) unit( 10px / @base-font-size-px, em );
+    background-color: @teal-mid-dark;
+    border-radius: 4px;
+    color: @white;
+    position: relative;
```

Comment:
I think this might be unnecessary?
```suggestion
```

Reply:
Removed in https://github.com/cfpb/consumerfinance.gov/pull/6038/commits/6aed924cf92ba897cce6715791d033a41c55bc7f